docs/nodes.md

```diff
@@ -2,9 +2,9 @@
 
 Modified from [comments in #3471](https://github.com/kubernetes-sigs/kubespray/issues/3471#issuecomment-530036084)
 
-## Limitation: Removal of first kube_control_plane and etcd-master
```

Comment:
Sorry for my unclear comment.
It is not necessary to remove master from etcd, because we just have a consensus to replace `master` of Kubernetes cluster with `control plane`, but don't have the one for etcd.

Reply:
+